Tests/AWSLambdaRuntimeCoreTests/Utils.swift

```diff
@@ -29,7 +30,14 @@ func runLambda(behavior: LambdaServerBehavior, factory: @escaping Lambda.Handler
     let runner = Lambda.Runner(eventLoop: eventLoopGroup.next(), configuration: configuration)
     let server = try MockLambdaServer(behavior: behavior).start().wait()
     defer { XCTAssertNoThrow(try server.stop().wait()) }
-    try runner.initialize(logger: logger, factory: factory).flatMap { handler in
+    
+    let serviceLifecycle = ServiceLifecycle(configuration: .init(shutdownSignal: [], installBacktrace: false))
+    defer {
+        serviceLifecycle.shutdown()
+        serviceLifecycle.wait()
+    }
```

Comment:
I guess we can register the server with lifecycle but maybe will make the test harder to reason about